config/internal/common/mlmd-envoy-dashboard-access-policy.yaml.tmpl

```diff
@@ -0,0 +1,24 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: ds-pipelines-envoy-sample
```

Comment:
name `sample` whouldn't be here, this should be parameterized to dspa name

Reply:
Done, updated to parameterize it.